src/devices/Is31fl3731/LedShimRgb28x1.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.I2c;
+using System.Threading;
+
+namespace Iot.Device.Display
+{
+    /// <summary>
+    /// Represents 28x1 RGB matrix, driven by an IS31FL3731 LED chip.
+    /// </summary>
+    // Product: https://shop.pimoroni.com/products/led-shim
+    public class LedShimRgb28x1 : Is31fl3731
+    {
+        /// <summary>
+        /// Initialize LED driver.
+        /// </summary>
+        /// <param name="i2cDevice">The <see cref="System.Device.I2c.I2cDevice"/> to create with.</param>
+        public LedShimRgb28x1(I2cDevice i2cDevice)
+        : base(i2cDevice, 28, 1)
+        {
+        }
+
+        /// <summary>
+        /// Default I2C address for device.
+        /// </summary>
+        public static new readonly int DefaultI2cAddress = 0x75;
+
+        /// <inheritdoc/>
+        public override int GetLedAddress(int x, int y) => (x, y) switch
+        {
+            // y == 0
+            (< 7, 0) => 118 - x,
+            (< 15, 0) => 141 - x,
+            (< 21, 0) => 106 + x,
+            (21, 0) => 15,
+            (_, 0) => x - 14,
+            // y == 1
+            (<2, 1) => 69 - x,
```

Comment:
you're missing some spaces in the following lines

Reply:
I'm not following. Can you elaborate?